Gemfile

```diff
@@ -49,9 +49,7 @@ group(:development, :test) do
   gem "yarjuf", "~> 2.0"
 
   # json-schema does not support windows, so omit it from the platforms list
-  # json-schema uses multi_json, but chokes with multi_json 1.7.9, so prefer 1.7.7
-  gem "multi_json", "1.7.7", :require => false, :platforms => [:ruby, :jruby]
-  gem "json-schema", "2.1.1", :require => false, :platforms => [:ruby, :jruby]
+  gem "json-schema", "~> 2.0", :require => false, :platforms => [:ruby, :jruby]
```

Comment:
I thought this was pinned to an earlier json-schema because newer versions depended on addressable, which requires ruby 2... but schema validation works using this PR, so 👍

Reply:
I see >= 1.9 for the most recent release of this gem on Rubygems.